README.md

```diff
@@ -6,7 +6,6 @@ A brief description of your project.
 
 - [Introduction](#introduction)
 - [Branches](#branches)
-- [Running Server](#Server)
 - [Committing](#committing)
 - [File Structure](#file-structure)
 
```

Comment:
Since we have our first route, we need this to be documented. Also, we should be ready to start writing a very basic unit test so console logs should no longer be needed. I still need to look at configuring and any other dependencies mocha has though.